git-work.sh

```diff
@@ -31,47 +31,46 @@ usage() {
   echo "Available subcommands are:"
   echo "   issue <id>              Start a new branch from issue <id>"
   echo "   commit <\"msg\">          Commit current work using <message>"
-  echo "   done <id>               Finish current work and merge"
-  echo "   push <master>           Push to the server"
+  echo "   done                    Finish current work and merge"
+  echo "   push                    Push to the server"
   echo "   release <tag> [<\"msg\">] Release a new tag called <tag> with <message>"
   echo "   pr <id>                 Create a branch from a pull-request"
 }
 
-if [ -z $1 ] || [ -z $2 ]; then
-  usage
-  exit 1
-fi
-
 case "$1" in
   # start a new branch
   issue)
+    [ -z $2 ] && ( usage && exit 1 )
     git checkout -b "$2"
     ;;
   # commit current work
   commit)
+    [ -z $2 ] && ( usage && exit 1 )
     git commit -am "$2"
     ;;
   # finish current work and merge
   done)
-    git checkout master
-    git merge "$2"
+    branch=${2:-$(git symbolic-ref --short HEAD)}
```

Comment:
This is not right. See the log below.

It is supposed to do:
1. switch to `master`
2. merge in `master`

The command `git work done` didn't merge with `master`.

```bash
> git-work-test (master)
$ git work issue 1
Switched to a new branch '1'

> git-work-test (1)
$ git status
On branch 1
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   foo.txt

no changes added to commit (use "git add" and/or "git commit -a")
> git-work-test (1)
$ git add .

>git-work-test (1)
$ git work commit "changed"
[1 6ad1ec8] changed
 1 file changed, 2 insertions(+), 1 deletion(-)

> git-work-test (1)
$ git work done
Already up-to-date.
```